### ISSUE TITLE
[Bug] TypeError: Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by alamothe** [berstend/puppeteer-extra#574]
**Describe the bug**

After upgrading `puppeteer-extra-plugin-stealth` from `2.6.7` to a new version, and installing `puppeteer-extra-plugin-user-data-dir`, I get the following when I try to run it:

```
/Users/alamothe/Projects/car-manager/login-service/node_modules/universalify/index.js:15
  }, 'name', { value: fn.name })
                         ^
TypeError: Cannot read property 'name' of undefined
    at exports.fromCallback (/Users/alamothe/Projects/car-manager/login-service/node_modules/universalify/index.js:15:26)
    at Object.<anonymous> (/Users/alamothe/Projects/car-manager/login-service/node_modules/fs-extra/lib/fs/index.js:57:27)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:14)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/Users/alamothe/Projects/car-manager/login-service/node_modules/fs-extra/lib/index.js:5:6)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
```

This happens on both MacOS and Windows machine. 

**Versions**

```
  System:
    OS: macOS 11.6
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 3.98 GB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.17.1 - /usr/local/opt/node@14/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.13 - /usr/local/opt/node@14/bin/npm
  npmPackages:
    puppeteer: 10.4.0 => 10.4.0 
    puppeteer-extra: 3.2.3 => 3.2.3 
    puppeteer-extra-plugin-recaptcha: 3.5.0 => 3.5.0 
    puppeteer-extra-plugin-stealth: 2.7.10 => 2.7.10 
    puppeteer-extra-plugin-user-data-dir: ^2.3.1 => 2.3.1 
```

**Comment by berstend** [comment]
Is it possible one of the dependencies conflicts with a version your project depends on as well? 🤔 

It'd be interesting to see if a fresh repo with the same versions and just the example code from the readme would yield this issue as well.

Alternatively the output of these commands would be interesting (specifically the package version numbers):
```bash
yarn why universalify
yarn why fs-extra
```

**Comment by peteriman** [comment]
Related to `fs.realpath.native` being undefined, causing import of `fs-extra` to throw error.

Patched in https://github.com/jprichardson/node-fs-extra/pull/953 and released in [`fs-extra@10.1.0`](https://github.com/jprichardson/node-fs-extra/blob/10.1.0/CHANGELOG.md).

**Comment by peteriman** [comment]
Related to `fs.realpath.native` being `undefined`, causing import of `fs-extra@10.0.0` to throw error.

Defensively patched in https://github.com/jprichardson/node-fs-extra/pull/953 and released in [`fs-extra@10.1.0`](https://github.com/jprichardson/node-fs-extra/blob/10.1.0/CHANGELOG.md).

But still, why is `fs.realpath.native` `undefined`?  That is the root cause to be investigated.

**Comment by peteriman** [comment]
Related to `fs.realpath.native` being `undefined`, causing import of `fs-extra@10.0.0` to throw error.

Defensively patched in https://github.com/jprichardson/node-fs-extra/pull/953 and released in [`fs-extra@10.1.0`](https://github.com/jprichardson/node-fs-extra/blob/10.1.0/CHANGELOG.md).

But still, why is `fs.realpath.native` `undefined`?  That is a root cause to be investigated.
